### ISSUE TITLE
get a linear classifier like svm or soft max into the transfer learning

### ISSUE THREAD

**Issue by cpadavis** [cpadavis/strongcnn#9]
The course people were against using random forests for this task. I do not totally understand why (another part of this task!) but it is worth creating another classifier just to see how it performs

**Comment by cpadavis** [comment]
did it! added svm and softmax. I think the trick was to use stochastic gradient descent from sklearn instead of the full linear svc, which tried to create some mondo matrix to compute the gradients
